runtime/queries/column_timeseries.go

```diff
@@ -385,11 +397,21 @@ func getFilterFromDimensionValuesFilter(
 		if notNulls {
 			inClause := escapedName + " " + prefix + " IN ("
 			for j, iv := range dv.In {
-				if _, ok := iv.Kind.(*structpb.Value_NullValue); !ok {
-					inClause += "'" + EscapeSingleQuotes(iv.GetStringValue()) + "'"
-					if j < len(dv.In)-1 {
-						inClause += ", "
-					}
+				switch iv.Kind.(type) {
+				case *structpb.Value_StringValue:
+
+					inClause += "'" + escapeSingleQuotes(iv.GetStringValue()) + "'"
```

Comment:
Let's also use `?` here and pass the string value in `args` – should be safer to let DB interpolate